webapp/src/pages/PlayPage/PlayPage.tsx

```diff
@@ -58,9 +108,46 @@ export default function PlayPage() {
     });
   }
 
+  if (!authUser) {
+    return <LandingPage />;
+  }
+
   return (
-    <SearchContextProvider fetchFn={mockFetchFn} maxEntries={ENTRIES_LIMIT}>
-      <MainTabTemplate dataMapper={mockedMapper} buttons={buttons} />
-    </SearchContextProvider>
+    <div className="play-page">
+      <div className="play-page-avatar">
+        <Link to={`${USER_ME_URL}`}>
+          <MediumAvatar
+            url={`${AVATAR_EP_URL}/${authUser.avatarId}`}
+            XCoordinate={authUser.avatarX}
+            YCoordinate={authUser.avatarY}
+          />
+        </Link>
+      </div>
+      <div className="play-page-search">
+        <div className="play-page-search-form">
+          <Input
+            variant={InputVariant.DARK}
+            iconVariant={IconVariant.SEARCH}
+            placeholder="search"
+            value={search}
+            name="search"
+            onChange={(e) => setSearch(e.target.value)}
+          />
+        </div>
+      </div>
+      <div className="play-page-games">
+        <RowsList rows={gameMapper()} />
```

Comment:
todo (in other PRs): We could replace the row list with a list of items similar to the score component.